client/js/hah-app/header.js

```diff
@@ -28,12 +26,7 @@ const styles = {
 		right: '0',
 		margin: 'auto',
 	},
-	authenticationButtonDiv: {
-		padding: '10px',
-		position: 'absolute',
-		top: '30px',
-		right: '30px'
-	},
+	authenticationButtonDiv: {},
```

Comment:
@bhavyaab If we're going to remove all the styles, we could also just remove this completely and remove the classNames for this style from above.

Reply:
It also looks like the logo style is not used anymore and can be removed...